### DISCUSSION TITLE
构建paddleocr的wheel包

### DISCUSSION THREAD

**Discussion by SongMingzhe** [PaddlePaddle/PaddleOCR#14296] [Q&A]
请问如何从PaddleOCR源码构建paddleocr的wheel包？

**Comment by GreatV** [comment]
看一下文档

**Comment by SWHL** [comment]
参见这个： https://github.com/PaddlePaddle/PaddleOCR/blob/main/.github/workflows/python-publish.yml

一般是clone项目到本地，直接执行：
```bash
python setup.py bdist_wheel
```

**Reply by TommyLee1202** [reply]
构建出来是不带ppocr和tools文件夹的，好像不太对

**Reply by SWHL** [reply]
参考这个：https://github.com/PaddlePaddle/PaddleOCR/blob/main/.github/workflows/python-publish.yml
